src/components/Quote.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import './Quote.css';
+
+const Quotes = () => {
+  const [data, setData] = useState('');
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      try {
+        const quote = await fetch('https://api.api-ninjas.com/v1/quotes?',
+          {
+            method: 'GET',
+            headers: {
+              'X-Api-Key': '8m1bnnM+p0GqxgD5kNoIgA==4OYoBSs5rh1rrEan',
+            },
+          });
+        const quoteObj = await quote.json();
+        setData(quoteObj[0].quote);
+      } catch (error) {
+        setError(error.message);
+      }
+    };
+    fetchQuote();
+  }, []);
```

Comment:
- [ ] The beauty and essence of useEffect the power to rerun when data changes and we actually expect data to change in this case, so we can actually watch when the data changes by adding both setData and setError in the empty array and when those functions change the data, the useEffect will rerun.